src/controllers/user-role-settings.controller.js

```diff
@@ -0,0 +1,34 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+/**
+ *
+ *
+ * @class UserRoleSettingsController
+ */
+class UserRoleSettingsController {
```

Comment:
Change this class to `User` or `Settings` controller. Gave feedback that files should not be created in isolation but they must be seen as a group or collection.

Reply:
@adesege  Ok solved now on by renaming all files related to this feature in both `controllers` and `swagger` and  `routes`